### ISSUE TITLE
Add docs for seeding RNG with seedRNG(int)

### ISSUE THREAD

**Issue by mtfishman** [ITensor/website#13]
As described in this support answer: http://www.itensor.org/support/866/initialize-the-random-seed-of-mps-sites

**Comment by emstoudenmire** [comment]
Yes, good to add, although shoudn’t this be posted to ITensor/ITensor issues, instead of ITensor/website?

**Comment by mtfishman** [comment]
Hmm I dunno, should we add docs issues there or here? I guess it is more consolidated if it is over there.

**Comment by emstoudenmire** [comment]
Oh, I see this is about docs. I thought you meant about adding a new function to ITensor that doesn't exist, which is why I was saying that.
